src/importexport/guitarpro/internal/importptb.cpp

```diff
@@ -703,7 +703,7 @@ void PowerTab::fillMeasure(tBeatList& elist, Measure* measure, int staff, std::v
                 note->setFret(n.value);
                 note->setString(n.str);
                 const StringData* sd = score->staff(staff)->part()->instrument()->stringData();
-                int k     = int(curTrack->infos[staff].strings.size()) - n.str - 1;
+                int k     = std::max(int(curTrack->infos[staff].strings.size()) - n.str - 1, 0);
```

Comment:
We used to use this check in previous version, but looks like it was deleted during some refactoring.